torch/_functorch/partitioners.py

```diff
@@ -455,6 +457,12 @@ def classify_nodes(joint_module):
         view_ops += [aten.view, aten.slice, aten.permute, aten.t, prims.broadcast_in_dim, aten.expand, aten.as_strided]
         # Natalia said that we should allow recomputing indexing :)
         default_recomputable_ops += [aten.index]
+
+        # this takes in a seed, so it's deterministic, and we can recompute it
+        default_recomputable_ops += [prims.inductor_random]
+
+        random_ops += [prims.inductor_seeds, prims.inductor_seed]
```

Comment:
We should probably recompute seeds? This might change rng behavior.

Reply:
This does prevent recomputation of seeds

Reply:
Ah, sorry, misssed it.